preload/preload.mk

```diff
@@ -2,8 +2,8 @@ ALL_TARGETS += preload/shellex_preload.so
 INSTALL_TARGETS += install-shellex_preload
 CLEAN_TARGETS += clean-shellex_preload
 
-SHELLEX_CFLAGS=-fPIC
-SHELLEX_PRELOAD_LDFLAGS=-shared
+SHELLEX_PRELOAD_LDFLAGS += -shared
+SHELLEX_PRELOAD_CFLAGS  += -fPIC
```

Comment:
Now, I acknowledge that this is incredibly nitpicky and there should just be an auto formatting for this. But line 2 and 3 don't have aligned operators, so lines 5 and 6 shouldn't either. Or both should. Either way. Consistency and all…